eng/Version.Details.xml

```diff
@@ -67,6 +67,10 @@
       <Uri>https://github.com/dotnet/aspnetcore</Uri>
       <Sha>19379d2de0cdcc8c89183bd8dbb847e145b16bd2</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.Test.ProjectTemplates.2.1" Version="1.0.2-beta4.20162.1">
```

Comment:
@ViktorHofer Add the same for 3.0 please. It is at the end of life, but the templates are still shipped in this, and we still build them in test-templates, so there is no reason to not update them automatically as well.

Reply:
Okay nvm. https://github.com/dotnet/core-sdk/issues/6807

Reply:
I created https://github.com/dotnet/core-sdk/issues/6807 to track removing them. I don't we should add a dependency for those if we plan to remove them.